packages/core-mobile/app/new/features/accountSettings/components/AcccountList.tsx

```diff
@@ -78,8 +78,8 @@ export const AccountList = (): React.JSX.Element => {
   const gotoAccountDetails = useCallback(
     (accountIndex: number): void => {
       navigate({
-        pathname: './accountDetails',
-        params: { accountIndex }
+        pathname: './accountSettings/account',
+        params: { accountIndex: accountIndex.toString() }
       })
```

Comment:
in the account settings, when the active account index is 1 or 2, I feel like we shouldn't animate. it feels a bit strange. what do you think?